cql/ManageSpecialPopulation.cql

```diff
@@ -721,7 +722,20 @@ define RecommendationForManagingPatientsAfterHysterectomy:
         date: Collate.DateOfMostRecentReport + 3 years,
         group: 'Hysterectomy (K.4)',
         details: {
-          'HPV-based testing (cotest or primary hrHPV test) is recommended every 3 years for 25 years.'
+          'HPV-based testing (cotest or primary hrHPV test) is recommended every 3 years for 25 years after hysterectomy is performed for treatment of histologic HSIL (CIN 2 or CIN 3).'
+        }
+      }
+    when ( // no hysterectomy date
+      SAR.AbsenceOrRemovalOfCervix
```

Comment:
```suggestion
      Rare.AbsenceOrRemovalOfCervix
```
This same function is defined in `ManageRareAbnormality`, which is already "include"ed in this CQL file. Bringing it in from `ManageRareAbnormality` means we don't have to add another dependency to this file, which should ultimately keep this file a little more lightweight when compiled.